containers/python/python_container.py

```diff
@@ -17,9 +19,22 @@ def load_predict_func(file_path):
         return cloudpickle.load(serialized_func_file)
 
 
+def load_modules(modules_list_path):
+    with open(modules_list_path, 'rb') as f:
+        module_fnames = pickle.load(f)
+        for m in module_fnames:
+            importlib.import_module(m)
```

Comment:
Ah, you're right -- looks like we don't need to import, just make the importable by adding their enclosing folder to sys.path.